### PR TITLE
Update doc to indicate ORC and Parquet zstd read support [skip ci]

### PR DESCRIPTION
**Pull request by sameerz**
Update compatibility.md to note read and write support for zstd for ORC and Parquet.  

Closes issue #6570

### CODE REVIEW

File: docs/compatibility.md
Comment (tgravescs) [inline docs/compatibility.md:370]: I didn't realize writing was supported but I see the cudf issue is resolved to support it, but I don't see that we have any tests for it.  Perhaps we should file an issue to enable tests?  Also note we still have https://github.com/NVIDIA/spark-rapids/issues/3037 open so we can probably close it.

Comment (tgravescs) [reply docs/compatibility.md:370]: I guess this is covered by https://github.com/NVIDIA/spark-rapids/pull/6362/files

Comment (jlowe) [reply docs/compatibility.md:370]: Agreed, there are some unanswered questions relating to output file sizes when using the ZSTD compression support, as the zstd block size used by the GPU doesn't match the default block size used by the CPU when writing Parquet+zstd.

Comment (sameerz) [reply docs/compatibility.md:370]: Would it be better to document that writing zstd for Parquet and ORC is experimental, and can be enabled  by setting LIBCUDF_NVCOMP_POLICY=ALWAYS ?  

Setting to draft until 6362 is merged.

Comment (tgravescs) [reply docs/compatibility.md:370]: personally I would almost just do the read support in this PR and the write support should be documented with the other PR.

Comment (jlowe) [reply docs/compatibility.md:370]: Agree with @tgravescs, I see no reason to speculate in this PR when we can (and should) update the docs in the PR that adds the functionality in question.  Only reason we're adding docs in this PR is that the doc updates were missed when ZSTD read support tests went in via #5898.

Comment (sameerz) [reply docs/compatibility.md]: Updated to only indicate zstd read support.

**Review by tgravescs — APPROVED** [review]

**Review by jlowe — APPROVED** [review]

**Comment by jlowe** [comment]
build

**Comment by jbrennan333** [comment]
Note that I have a WIP PR up to add support for parquet/orc write compression in spark-rapids, but because the feature is still experimental, we may want to hold off on enabling it.  See [this comment](https://github.com/NVIDIA/spark-rapids/pull/6362#issuecomment-1252959031).

**Pull request merged**
